recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "2.2.27.6" %}
+{% set version = "2.2.27.27" %}
 
 package:
   name: udunits2
   version: {{ version }}
 
 source:
   url: https://github.com/Unidata/UDUNITS-2/archive/v{{ version }}.tar.gz
-  sha256: 74fd7fb3764ce2821870fa93e66671b7069a0c971513bf1904c6b053a4a55ed1
+  sha256: 6c65b3097ef38b94c01dfede8904346c1439de9704b9bae7deea72bd5143a7bb
 
 build:
-  number: 1001
+  number: 0
   detect_binary_files_with_prefix: true
-  skip: True  # [win and vc<14]
+  skip: True  # [win]
```

Comment:
Let's not skip Windows all together. I'd rather merge a PR with a failing Windows CI and open an issue to follow up than just remove the problem from sight.

Reply:
Ok, I will reenable the windows build. I thought a failing CI would be a hard blocker, but I agree with this approach.
Note, however, that upstream seems unable to build on Windows, see Unidata/UDUNITS-2#96, Unidata/UDUNITS-2#87.